src/test/java/seedu/trackermon/commons/core/ConfigTest.java

```diff
@@ -9,8 +9,14 @@
 
 import org.junit.jupiter.api.Test;
 
+/**
+ * Contains integration tests for {@code Config}.
+ */
 public class ConfigTest {
 
+    /**
+     * Test method to test the string of a default {@code Config} object.
```

Comment:
Change this to `Tests the string of a default {@code Config} object.` ?

Reply:
The comments look good, but just to need to change most of them to start from a noun to a verb.